.github/workflows/test-deploy-main.yml

```diff
@@ -325,3 +325,17 @@ jobs:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
       - name: Publish to next tag
         run: yarn publish:next
+
+  build-test: 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Open canary folder
+        run: cd canary/
+      - name: Setup canary folder against @next
+        run: yarn setup:next 
+      - name: Run yarn install on each sample app
+        run: yarn install
+      - name: Run yarn build on each sample app
+        run: yarn build
```

Comment:
Since this is another build step, the `node_modules` in the root of the mono repo will not exist correct? That way we don't have any issues with missing dependencies being found higher in the tree?

Reply:
Right! Everything is happening from `canary/` folder.